spacy/cli/_util.py

```diff
@@ -439,3 +440,12 @@ def string_to_list(value: str, intify: bool = False) -> Union[List[str], List[in
             p = int(p)
         result.append(p)
     return result
+
+
+def set_gpu_allocator(allocator):
+    if allocator == "pytorch":
+        use_pytorch_for_gpu_memory()
+    elif allocator == "tensorflow":
+        use_tensorflow_for_gpu_memory()
+    else:
+        msg.warn(f"Ignored unknown 'gpu_allocator' argument: '{allocator}'")
```

Comment:
should this be here, or in the main `util` class? Perhaps users will want to call this from within their own code?

Reply:
Should it be in Thinc instead?